src/node_crypto_clienthello.cc

```diff
@@ -146,7 +149,8 @@ void ClientHelloParser::ParseExtension(const uint16_t type,
       ocsp_request_ = 1;
       break;
     case kTLSSessionTicket:
-      tls_ticket_size_ = len;
+      // TBD POSSIBLE DATA LOSS:
+      tls_ticket_size_ = static_cast<uint16_t>(len & kTLSTicketSizeMask);
```

Comment:
@indutny Is there a reason `tls_ticket_size_` is of type uint16_t instead of size_t?